pkg/liqo-controller-manager/virtualNode-controller/suite_test.go

```diff
@@ -128,6 +129,26 @@ var _ = BeforeSuite(func(done Done) {
 	k8sClient = k8sManager.GetClient()
 	Expect(k8sClient).ToNot(BeNil())
 
+	nms = &mapsv1alpha1.NamespaceMapList{}
+	technicalNamespace = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: liqoconst.TechnicalNamespace,
+		},
+	}
+
```

Comment:
Can you add some By to specify what are you testing? The test-case description is very limited